querybook/server/env.py

```diff
@@ -14,9 +14,9 @@ class MissingConfigException(Exception):
     pass
 
 
-def get_env_config(name, optional=True):
+def get_env_config(name, optional=True, default=None):
```

Comment:
for default values, can you put it in querybook_default_config.yaml? there is no need to put defaults in env.py

Reply:
done!